x-pack/plugins/apm/server/lib/metrics/by_agent/default.ts

```diff
@@ -13,8 +13,8 @@ export async function getDefaultMetricsCharts(
   serviceName: string
 ) {
   const charts = await Promise.all([
-    getCPUChartData(setup, serviceName),
-    getMemoryChartData(setup, serviceName),
+    getCPUChartData({ setup, serviceName }),
+    getMemoryChartData({ setup, serviceName }),
```

Comment:
The server side changes from here and down are purely stylistic (destructured arguments)